spec/helpers/override_helper_spec.rb

```diff
@@ -21,15 +21,26 @@
 
   describe '#complex_reason_label' do
     context 'when a prison POM' do
-      let(:subject) { OffenderPresenter.new(OpenStruct.new(immigration_case?: true), nil) }
+      # Immigration cases are always recommended to Prison POMs
```

Comment:
Could this be expressed with `describe`/`context`/`it` naming instead? I worry that comments won't stay up-to-date as the code changes further, which might confuse future readers.

Reply:
We've updated the comment to be more specific